src/main/java/com/pirimid/cryptotrade/helper/exchange/gemini/dto/request/CreateOrderRequest.java

```diff
@@ -0,0 +1,27 @@
+package com.pirimid.cryptotrade.helper.exchange.gemini.dto.request;
+
```

Comment:
this DTO is used to send request to exchange. right?

Reply:
Yes it is for Gemini only